include/picongpu/simulation/control/DomainAdjuster.hpp

```diff
@@ -81,9 +81,16 @@ namespace picongpu
 
             for(uint32_t d = 0; d < simDim; ++d)
             {
+                // doc-include-start: automatic-grid-adjustment
+                // The checks are performed for each dimension `d` individually.
+                // If not, the size in that dimension is rounded up to the nearest multiple:
                 multipleOfSuperCell(d);
+                // We need at least one CORE cell and one GUARD cell per MPI rank on either side. If not, this
```

Comment:
`We need at least one CORE and two GUARD supercell per MPI rank.`

The former text could be misinterpreted that two CORE supercells are required. I changed also cell to supercell.

Reply:
I hope I've found the best of both worlds. Done.